extensions/src/ACRE2Arma/signal/models/Arcade.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Arcade.hpp"
+#include <glm/geometric.hpp>
+
+acre::signal::model::Arcade::Arcade() : SignalModel()
+{
```

Comment:
> Opening braces should always be on the same line as their block identifier.

Reply:
Done it in new files. Changes in older files will be done in separate PRs